deploy/roles/r-studio-server/tasks/main.yml

```diff
@@ -40,3 +41,52 @@
   when: update_rstudio
   tags:
     - rstudio-server
+
+- name: Install pip (driver dependency)
+  yum:
+    name: python-pip
+    state: present
+  when: install_pro_drivers
+  tags:
+    - rstudio-server
+
+- name: pip install pexpect (driver dependency)
+  pip:
+    name: pexpect
+    executable: /usr/bin/pip
+  when: install_pro_drivers
+  tags:
+    - rstudio-server
+
+- name: Create RStudioPro driver folder
+  file:
+    path: "{{ rstudio_pro_driver_install_folder }}"
+    state: directory
+    mode: a+rwx
```

Comment:
@stirlingv we just need `a+rx`, no `w` since we don't want folks changing anything!